haskell/cabal.bzl

```diff
@@ -243,3 +243,225 @@ However, using a plain `haskell_library` sometimes leads to better
 build times, and does not require drafting a `.cabal` file.
 
 """
+
+def _chop_version(name):
+    """Remove any version component from the given package name."""
+    return name.rpartition("-")[0]
+
+# Temporary hardcoded list of core libraries. This will no longer be
+# necessary once Stack 2.0 is released.
+#
+# TODO remove this list and replace it with Stack's --global-hints
+# mechanism.
+_CORE_PACKAGES = [
+    "Cabal",
+    "array",
+    "base",
+    "binary",
+    "bytestring",
+    "containers",
+    "deepseq",
+    "directory",
+    "filepath",
+    "ghc",
+    "ghc-boot",
+    "ghc-boot-th",
+    "ghc-compact",
+    "ghc-heap",
+    "ghc-prim",
+    "ghci",
+    "haskeline",
+    "hpc",
+    "integer-gmp",
+    "libiserv",
+    "mtl",
+    "parsec",
+    "pretty",
+    "process",
+    "rts",
+    "stm",
+    "template-haskell",
+    "terminfo",
+    "text",
+    "time",
+    "transformers",
+    "unix",
+    "xhtml",
```

Comment:
Interestingly `hazel/packages.bzl` does not list `xhtml` as a core-package. Did this change between stackage releases?

Reply:
xhtml has been a core library for a long time. Probably an omission in Hazel. Or maybe an omission in Stackage.